### PR TITLE
Miscellaneous logging and clean up.

### PR DESCRIPTION
**Pull request by rohinb2**

### CODE REVIEW

**Comment by rohinb2** [comment]
Current dependencies on/for this PR:
* main
  * **PR #154** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/154?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #155** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/155?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #156** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/156?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/run-house/runhouse/154?utm_source=stack-comment).

File: runhouse/rns/rns_client.py [hunk 1]
```diff
@@ -364,8 +364,6 @@ def save_config(self, resource, overwrite: bool = True):
                 f"Resource {rns_address} already exists and overwrite is False."
             )
 
-        config["name"] = rns_address
```
Comment (rohinb2) [inline runhouse/rns/rns_client.py:367]: In `resource.config_for_rns`, which is called above, this is automatically set. This line is redundant.

File: runhouse/resources/resource.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ def __init__(
         if name is not None:
             # TODO validate that name complies with a simple regex
             if name.startswith("/builtins/"):
-                name = name[10:]
+                name = name[len("/builtins/") :]
```
Comment (joseph-zhong) [inline runhouse/resources/resource.py:53]: turn this into a varibale

**Review by dongreenberg — APPROVED** [review]

**Comment by rohinb2** [comment]
### Merge activity

* **Nov 20, 7:44 AM**: @rohinb2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/run-house/runhouse/154).

**Pull request merged**